SP-1648: add a log of major RSP updates

MelissaGraham opened the pull request:
Try out a log of major updates for RSP.

jonathansick reviewed (commented):
First, a favour. When you make a pull request, it'd be helpful for the rest of the team to have these PRs appropriately titled. The usual format we have is `SP-1234: Summary`. This is especially helpful to stakeholders like the RSP team that outside the CST, but are also operating in this repo.

The "change log" here is under user guides. My suggestion is that we move this to the root of the site, `/updates` and put a card pointing to this on the homepage. You can also put a link to this from the User Guides still. But the main thing is I'd recommend keeping it out of the User Guides section. This is to:

- improve the visibility of the change log
- it's content type is fundamentally different from the user guides
- It also provides a place in the hierarchy to include @frossie's roadmap page, i.e. `/updates/roadmap.html`.

Also as an FYI, down the road (in the data releases era) I want to have an RSS feed of these platform updates and also start pushing platform updates through the user notification system.

MelissaGraham commented:
Thanks @jonathansick  -- agree on all suggestions and incorporated them all in one commit.

 - changed PR title to "SP-1648: add a log of major RSP updates"
 - removed /guides/log folder
 - removed log section from /guides/index.rst
 - added /updates folder
 - renamed subsection from "Log of major updates" to just "Updates"
 - removed environment fork; entries can specify environment where applicable
 - text changes all made

Still have some build failures to figure out but I'll work on that.

MelissaGraham commented on docs/guides/log/index.rst:
```diff
@@ -0,0 +1,13 @@
+####################
+Log of major updates
+####################
```
Changed to "Updates" and use of top-level folder /updates.

MelissaGraham commented on docs/guides/log/index.rst:
```diff
@@ -0,0 +1,13 @@
+.. jinja:: rsp
+
+   {% if env.is_primary %}
+   .. include:: log.primary.in.rst
```
That makes sense to me -- removed the environment forking. There is only /updates/index.rst now, and no "primary" file.

MelissaGraham commented on docs/guides/log/log.primary.in.rst:
```diff
@@ -0,0 +1,27 @@
+Mar 6 2025
+==========
```
Adopted full month format.

MelissaGraham commented on docs/guides/log/log.primary.in.rst:
```diff
@@ -0,0 +1,27 @@
+
+
+Action items
+------------
```
Went with "Actions required" as the sub-section header.

MelissaGraham commented on docs/guides/log/log.primary.in.rst:
```diff
@@ -0,0 +1,27 @@
+Action items
+------------
+
+Any user-installed python packages with version 3.11 may need to be re-installed.
```
Adopted the suggested wording.

MelissaGraham commented on docs/guides/log/log.primary.in.rst:
```diff
@@ -0,0 +1,27 @@
+
+Any user-installed python packages with version 3.11 may need to be re-installed.
+
+Remove old versions of the tutorial notebooks from the old, write-only directory, in order to save space and avoid confusion.
```
Changed from period to colon.

MelissaGraham commented on docs/guides/log/log.primary.in.rst:
```diff
@@ -0,0 +1,27 @@
+==========
+
+DP0.2 images are now being served from their new location US Data Facility at SLAC (instead of Google Cloud).
+This change is transparent to users, except for a potential higher latency in image access (10-30%); a fix is in development.
```
Added percent sign.
